internal/dao/port_forwarder.go

```diff
@@ -176,6 +177,16 @@ func (p *PortForwarder) forwardPorts(method string, url *url.URL, addr, portMap
 	}
 	dialer := spdy.NewDialer(upgrader, &http.Client{Transport: transport, Timeout: defaultTimeout}, method, url)
 
+	tunnelingDialer, err := portforward.NewSPDYOverWebsocketDialer(url, cfg)
```

Comment:
Aren't websockets support in k8s gated? I think it might be best to check first if that feature is enabled and then fallback. I could be wrong here but volunteering the fallback right out of the gate could introduce additional delays initializing the connection.

Reply:
in 1.31 it's promoted to beta and enabled by default. https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.31.md?plain=1#L587

Reply:
Ah! Thank you for the fwd! I think best to pair up with kubectl and check `PORT_FORWARD_WEBSOCKETS`.

Reply:
Thanks for the reply! 
I get that, but if this feature is a dud or causing additional lag for the web socket call to time out,
I think it's best to give users the choice to turn this off client side if warranted.